eventuals/os.h

```diff
@@ -58,8 +61,83 @@ inline void SetAffinity(std::thread& thread, const size_t cpu) {
           &cpuset),
       0);
 }
+
+
 #endif // __MACH__
 
+static inline void CheckSufficientStackSpace(const size_t composed_size) {
```

Comment:
Let's add `inline Bytes GetStackSize()` and `inline void* GetStackStart()` and `inline void* GetStackEnd()` and `inline Bytes GetStackAvailable()`.